pkg/converter/sloppy_file.go

```diff
@@ -172,6 +188,35 @@ func (sf *SloppyFile) convertCommand(cmd []interface{}) *string {
 	return &str
 }
 
+var resourceMemRegex = regexp.MustCompile(`^(\d+)([bkmgBKMG]){1}$`)
```

Comment:
I'm curious: Is the `{1}` part needed? Looks like removing it would result in the same expression. The `([...])` block should already match only one character.